### DISCUSSION TITLE
q-uploader failed  401 unauthorized

### DISCUSSION THREAD

**Discussion by badblli** [quasarframework/quasar#15108] [General - Components / Directives / etc]
In the q-uploader component, an unauthorized error is returned even though I enter the user token and required field under the header.

```
 <q-uploader
                label="Custom header"
                style="margin-top: 10px; width: 100%"
                :accept="selectedType"
                :disable="selectedType === null"
                method="post"
                auto-upload
                url="http://test.xxx.api/upload"
                :headers="[
                  {
                    Authorization: this.userStore.Token,
                    GlobalCompanyID: '0',
                  },
                ]"
              >
```

![image](https://user-images.githubusercontent.com/75008512/207621048-72d2737b-9949-421c-9db9-c9741d4429b8.png)

Why is GlobalCompanyID being sent as null even though I specify it in the header?

**Comment by metalsadman** [comment]
don't use `this` in the template. `this.userStore.Token,`

**Reply by badblli** [reply]
I removed the ` [ ]`  in the header and I no longer get the 401 unauthorized error. As you said, I removed `this`  in the template. but this time the upload loading icon spinning and I can't see any post process in the network section of my browser. what could be the reason for this?
